pandas/core/groupby/generic.py

```diff
@@ -144,8 +147,34 @@ def _cython_agg_blocks(self, how, alt=None, numeric_only=True,
         return new_items, new_blocks
 
     def aggregate(self, func, *args, **kwargs):
-
         _level = kwargs.pop('_level', None)
+
+        relabeling = func is None and _is_multi_agg_with_relabel(**kwargs)
+        if relabeling:
+            if not PY36:
+                kwargs = OrderedDict(sorted(kwargs.items()))
+
+            # Normalize the aggregation functions as Dict[column, List[func]],
+            # process normally, then fixup the names.
+            # TODO(Py35): When we drop python 3.5, change this to
```

Comment:
Hmm couldn't we just do this now since `order` keeps track of that for us?

Reply:
I don't recall if I checked, but I thought we needed this to ensure that the order of the arguments is respected in `_python_agg_general`.